comments and bullshit

```diff
@@ -0,0 +1,51 @@
+{ GameCount: 2,
```

Comment:
Nice way of setting up for testing.

(I _think_ there's a hotkey combination that will short out the JSON format here, but I can't find it in Visual Studio Code, so it might be missing :( )